docker-compose.yml

```diff
@@ -17,6 +17,8 @@ services:
     volumes:
       - ./Website/config/php/php.ini:/usr/local/etc/php/php.ini
       - ./Website/htdocs:/var/www/html
+    command: [ "sh", "-c", "cd /var/www/html/mpmanager && php composer.phar install" ]
```

Comment:
Doesn't this overwrite the `CMD` from the underlying Dockerfile `CMD ["apache2-foreground"]` ?https://hub.docker.com/layers/library/php/8.2-apache/images/sha256-39d43cc32ea91e96fa7333cd52f67b6d91c8c4b22130443db0b1ac5f26b7c738?context=explore ?

Reply:
@dmohns oh very good catch thank you for this 🙏 . I totally missed that one.